examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.2_identity_security/identity_security.ipynb

```diff
@@ -0,0 +1,210 @@
+    "* FLARE Console (previously called Admin Client) kits are sent to the user(s)\n",
+    "\n",
+    "To ensure the integrity of the Startup Kit, each file in the kit is signed by the Root CA.\n",
+    "\n",
+    "Each Startup Kit also contains a “start.sh” file, which can be used to properly start the NVFLARE application.\n",
+    "\n",
+    "Once started, the Client tries to establish a mutually-authenticated TLS connection with the Server, using the PKI credentials in its Startup Kits. This is possible only if the client and the server both have the correct Startup Kits.\n",
+    "\n",
+    "Similarly, when a user tries to operate the NVFLARE system with the Admin Client app, the admin client tries to establish a mutually-authenticated TLS connection with the Server, using the PKI credentials in its Startup Kits. This is possible only if the admin client and the server both have the correct Startup Kits. The admin user also must enter his/her assigned user name correctly.\n",
+    "\n",
+    "With Release 2.6.0, we also supports the normal TLS ( in additional to mutal TLS). \n",
+    "\n",
+    "\n",
+    "The security of the system comes from the PKI credentials in the Startup Kits. As you can see, this mechanism involves manual processing and human interactions for Startup Kit distribution, and hence the identity security of the system depends on the trust of the involved people. To minimize security risk, we recommend that people involved follow these best practice guidelines:\n",
+    "\n",
+    "The Project Admin, who is responsible for the provisioning process of the study, should protect the study’s configuration files and store created Startup Kits securely.\n",
+    "\n",
+    "When distributing Startup Kits, the Project Admin should use trusted communication methods, and never send passwords of the Startup Kits in the same communication. It is preferred to send the Kits and passwords with different communication methods.\n",
+    "\n",
+    "Org Admin and users must protect their Startup Kits and only use them for intended purposes.\n",
+    "\n",
+    ">  Note\n",
+    "The provisioning tool tries to use the strongest cryptography suites possible when generating the PKI credentials. All of the certificates are compliant with the X.509 standard. All private keys are generated with a size of 2048-bits. The backend is openssl 1.1.1f, released on March 31, 2020, with no known CVE. All certificates expire within 360 days.\n",
+    "\n",
+    "> NVFlare Dashboard is a website that supports user and site registration. Users will be able to download their Startup Kits (and other artifacts) from the website.\n",
+    "\n",
+    "Let's take a look above in action\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, let's generate a default startup kits using POC mode. POC internally call provision to generate the default startup kits"
```

Comment:
-> "using the POC (Proof-Of-Concept) command. The POC command internally calls the provision tool to..."